frontend/src/components/pages/CampCreation/CampCreationFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Button, Flex, Spacer } from "@chakra-ui/react";
+import CampCreationPages from "./CampCreationPages";
+
+export type CampCreationFooterProps = {
+  currentStep: CampCreationPages;
+  isCurrentStepCompleted: boolean;
+  handleStepNavigation: (stepsToMove: number) => void;
+};
+
+const CampCreationFooter = ({
+  currentStep,
+  isCurrentStepCompleted,
+  handleStepNavigation,
+}: CampCreationFooterProps): React.ReactElement => {
+  const onNextStep = () => {
+    if (isCurrentStepCompleted) {
```

Comment:
I am not sure if I understand the logic here completely. When you're on the registrationFormPage step (the last step) and you click next, wouldn't you go to the next step (which doesn't exist)?

Reply:
I guess the function is created in a way that it would do nothing if we increment past the last step? Also, wouldn't isCurrentStepCompleted always be true since the button would be disabled otherwise?

Reply:
Looks like `onNextStep()` will call the index's `handleStepNavigation()`, and you're right that doesn't do anything if we're trying to move to a page that doesn't exist i.e. moving forward after the last page. After we implement the publish functionality, we can pass in a method to call the publish function as a prop to the footer.

Regarding your `isCurrentStepCompleted` comment, yeah you're right the if statement here is unneccesary, I removed it.